tests/NuGet.Services.BasicSearchTests/NuGet.Services.BasicSearchTests.csproj

```diff
@@ -131,7 +131,7 @@
       <Version>2.28.0</Version>
     </PackageReference>
     <PackageReference Include="xunit">
-      <Version>2.1.0</Version>
+      <Version>2.4.1</Version>
```

Comment:
This seems duplicate with https://github.com/NuGet/NuGet.Services.Metadata/pull/523

Reply:
Hence the separate PR ;) merged with dev.